ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-commands.sh

```diff
@@ -153,16 +140,17 @@ ibm_cloud_cis_crn = "${IBMCLOUD_CIS_CRN}"
 ibm_cloud_tgw              = "${workspace_name}-tg"
 EOF
 
-  cp "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/var-mac-upi.tfvars "${SHARED_DIR}"/var-mac-upi.tfvars
-  cat "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/var-mac-upi.tfvars
+  cp "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/var-multi-arch-upi.tfvars "${SHARED_DIR}"/var-multi-arch-upi.tfvars
+  cat "${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/var-multi-arch-upi.tfvars
```

Comment:
```suggestion
  echo "UPI TFVARS created: ${IBMCLOUD_HOME_FOLDER}"/ocp-install-dir/var-multi-arch-upi.tfvars
```
Let's not `cat` it out.